src/libsync/propagateremotemove.h

```diff
@@ -25,8 +25,12 @@ namespace OCC {
 class MoveJob : public AbstractNetworkJob {
     Q_OBJECT
     const QString _destination;
+    const QUrl _url; // Only used (instead of path) when the constructor taking an URL is used
+    QMap<QByteArray, QByteArray> _extraHeaders;
```

Comment:
I'm wondering if this should somehow be in AbstractNetworkJob? but maybe it can't if the subclasses are the ones instaciating the reply..

so not important.